includes/admin/views/omise-page-settings.php

```diff
@@ -1,6 +1,24 @@
 <div class="wrap omise">
+	<style>
+		.omise-notice-testmode {
+			background: #ffce00;
+			color: #575D66;
+			border: 1px solid #efc200;
+			border-left-width: 4px;
+		}
+	</style>
 	<h1><?php echo $title; ?></h1>
+
+	<?php $page->display_messages(); ?>
+
+	<?php if ( 'yes' === $settings['sandbox'] && $settings['account_email'] ) : ?>
```

Comment:
Why check for `account_email` if the main thing we're concerned about is the value of `sandbox`?

Reply:
Because the _sandbox_'s value is `yes` by default. Adding `$settings['account_email']` to make this bar display only when an Omise account has been set and the test mode is enabled.

Reply:
I suppose, but it's still true that no actual payment can be made when you haven't connected an account, right? 😏

Reply:
Hmm, good point actually